docs/project/linux-performance-tracing.md

```diff
@@ -165,28 +165,22 @@ symbols for these when it converts its data, but ONLY if the symbols for these n
 the library they are for).
 
 There is a global command called [dotnet symbol](https://github.com/dotnet/symstore/blob/master/src/dotnet-symbol/README.md#symbol-downloader-dotnet-cli-extension) which does this.   This tool was mostly desiged to download symbols
-for debugging, but it works for perfcollect as well.  There are three steps to getting the symbols
+for debugging, but it works for perfcollect as well.  There are two steps to getting the symbols:
 
-   1. Install dotnet symbol
+   1. Install dotnet symbol.
    2. Download the symbols.
-   3. Copy the symbols to the correct place
 
 To install dotnet symbol issue the command
 ```
      dotnet tool install -g dotnet-symbol
 ```
-With that installed download the symbols to a local directory.  if your installed version of the .NET Core runtime is
-2.1.0 the command to do this is
-```
-    mkdir mySymbols
-    dotnet symbol --symbols --output mySymbols  /usr/share/dotnet/shared/Microsoft.NETCore.App/2.1.0/lib*.so
-```
-Now all the symbols for those native dlls are in mySymbols.   You then have to copy them (as super user next to the
-dlls that they are for.
+
+To download symbols for all native libraries and store them next to them:
+
```

Comment:
I think that this is a good improvement.  I also recommend that you make it clear that this will download symbols for the .NET runtime/framework as well as any other installed frameworks (e.g. ASP.NET), and give folks an idea of how to limit to just one type of framework (e.g. Microsoft.NETCore.App).